engine/pokemon/learn.asm

```diff
@@ -102,25 +102,25 @@ LearnMove:
 	jp .learned
 
 .cancel
-	ld hl, Text_StopLearning ; Stop learning <MOVE>?
+	ld hl, MoveAskStopLearningText ; Stop learning <MOVE>?
 	call PrintText
 	call YesNoBox
 	jp c, .loop
 
-	ld hl, Text_DidNotLearn ; <MON> did not learn <MOVE>.
+	ld hl, MoveDidNotLearnText ; <MON> did not learn <MOVE>.
 	call PrintText
 	ld b, 0
 	ret
 
 .learned
-	ld hl, Text_LearnedMove ; <MON> learned <MOVE>!
+	ld hl, MoveLearnedText ; <MON> learned <MOVE>!
```

Comment:
LearnedMoveText

Reply:
There's already a `LearnedMoveText`, used in `engine/battle/move_effects/mimic.asm`. Renaming that one to `MimicLearnedMoveText`.